Makefile

```diff
@@ -173,5 +174,20 @@ fabric:
 		rm -rf Frameworks/Fabric; \
 	fi
 
+stripe:
```

Comment:
Now that we're doing this download and unzip dance for 3 frameworks perhaps it makes sense to extract this into a shell script that takes a URL and a path? Pity that the OpenTok one is a tar ball and not a zip though 🤔

Reply:
@justinswart noted! I've added a card to our backlog to [refactor the Makefile 3rd party downloads into a bash script](https://trello.com/c/GH1NIT7F/464-update-makefile-for-frameworks-to-use-shared-bash-script)